build.sbt

```
@@ -93,9 +93,10 @@ lazy val tyrian =
     .jsSettings(
       commonJsSettings,
       libraryDependencies ++= Seq(
-        "org.scala-js"  %%% "scalajs-dom"        % Dependancies.scalajsDomVersion,
-        "org.typelevel" %%% "cats-effect-kernel" % Dependancies.catsEffect,
-        "co.fs2"        %%% "fs2-core"           % Dependancies.fs2
+        "org.scala-js"      %%% "scalajs-dom"        % Dependancies.scalajsDomVersion,
+        "org.typelevel"     %%% "cats-effect-kernel" % Dependancies.catsEffect,
+        "co.fs2"            %%% "fs2-core"           % Dependancies.fs2,
+        "com.github.buntec" %%% "scala-js-snabbdom"  % "0.0-29f78bf-SNAPSHOT"
```

```suggestion
        "io.github.buntec" %%% "scala-js-snabbdom"  % "0.0-1c017a3-SNAPSHOT"
```

You'll also need to add:
```scala
resolvers +=
  "Sonatype S01 OSS Snapshots" at "https://s01.oss.sonatype.org/content/repositories/snapshots"
```

---

That is not [published](https://s01.oss.sonatype.org/content/repositories/snapshots/com/github/) it seems, but I'll try a local publish again.

---

The group id changed `com.github.buntec` -> `io.github.buntec`.

https://s01.oss.sonatype.org/content/repositories/snapshots/io/github/buntec/scala-js-snabbdom_sjs1_2.13/0.0-1c017a3-SNAPSHOT/

---

Sorry - my bad - you changed the group to `io`...

---

Updated, that does indeed appear to fix the issues. 😄